src/playbacks/html5_video/html5_video.js

```diff
@@ -62,19 +65,37 @@ export default class HTML5Video extends Playback {
     return this.el.ended
   }
 
+  /**
+   * Determine if the playback is having to buffer in order for
+   * playback to be smooth.
+   * This is related to the PLAYBACK_BUFFERING and PLAYBACK_BUFFERFULL events
+   * @property buffering
+   * @type Boolean
+   */
+  get buffering() {
+    return this.bufferingState
+  }
+
   constructor(options) {
     super(options)
+    this.loadStarted = false
+    this.playheadMoving = false
+    this.playheadMovingTimeOnCheck = null
+    this.playheadMovingTimer = setInterval(this.determineIfPlayheadMoving.bind(this), 500)
```

Comment:
It was actually needed here because the scope of the `setInterval` callback isn't the object.